File: app/src/main/java/co/tpcreative/supersafe/common/response/SyncResponse.java
package co.tpcreative.supersafe.common.response;

import java.io.Serializable;
import java.util.List;

import co.tpcreative.supersafe.common.api.response.BaseResponse;
import co.tpcreative.supersafe.common.entities.ItemEntity;
import co.tpcreative.supersafe.common.entities.MainCategoryEntity;

public class SyncResponse extends BaseResponse implements Serializable {
    public List<ItemEntity>files;
    public List<MainCategoryEntity>listCategories;
}

File: app/src/main/java/co/tpcreative/supersafe/common/response/SyncResponse.kt
package co.tpcreative.supersafe.common.response
import co.tpcreative.supersafe.common.api.response.BaseResponse
import co.tpcreative.supersafe.model.ItemModel
import co.tpcreative.supersafe.model.MainCategoryModel
import java.io.Serializable

class SyncResponse : BaseResponse(), Serializable {
    var files: MutableList<ItemModel?>? = null
    var listCategories: MutableList<MainCategoryModel?>? = null
}